R/make_data_imp.R

```diff
@@ -31,7 +32,7 @@ make_data_imp <- function(data, n_imps = 3) {
 
   # Run imps with better settings
   future_cores <- parallel::detectCores() - 1
-  if(future_cores < n_imps) future_cores <- n_imps
+  if(future_cores > n_imps) future_cores <- n_imps
 
```

Comment:
What about making this simpler:

```suggestion
future_cores <- min(parallel::detectCores() - 1, n_imps)
```

Reply:
Looks good, weirdly wouldn't let me commit this suggestion so have pushed it manually.